docs/node-operators/hardforks.mdx

```diff
@@ -9,39 +16,44 @@ A hard fork occurs when there are major changes to the core protocol that requir
 
 ## Overview
 
-A fork in a blockchain happens when the rules that determine which blocks are valid or invalid change. There are two types of forks: hard forks and soft forks.
+A fork in a blockchain happens when the rules that determine which blocks are valid or invalid change. 
 
-A soft fork occurs when the new rules are still compatible with the old rules. This means users can still run an old version of the node software and still connect to the network.
+There are two types of forks: soft forks and hard forks.
 
-A hard fork means the old chain is incompatible with the new chain. An example of a hard fork is the Bitcoin Cash network, which used the original Bitcoin blockchain and then hard forked to a new network that is incompatible. As seen in this example, after a hard fork it is possible for two chains to exist. To avoid splitting the chain it is important that every node on the network updates to the latest software.
+- A soft fork occurs when the new rules are still compatible with the old rules. This means users can run an old version of the node software and still connect to the network.
+- A hard fork means the old chain is incompatible with the new chain. After a hard fork it is possible for two chains to exist. To avoid splitting the chain it is important that every node on the network updates to the latest software. For example, Bitcoin Cash is considered to be the most successful hard fork of the primary cryptocurrency.
```

Comment:
This note about Bitcoin Cash is misleading. That was a hard fork, but it was a hard fork *away* from Bitcoin. The kind of hard forks we are discussing here are the kind of hard forks that happen within the Mina blockchain. If we want an analogy, we should point to something like the ETH2 hard fork, although even that is a bit different in scope.